satellite/satellitedb/overlaycache.go

```diff
@@ -509,6 +509,31 @@ func (cache *overlaycache) KnownUnreliableOrOffline(ctx context.Context, criteri
 	return badNodes, nil
 }
 
+// Reliable returns all reliable nodes.
+func (cache *overlaycache) Reliable(ctx context.Context, criteria *overlay.NodeCriteria) (nodes storj.NodeIDList, err error) {
+	// get reliable and online nodes
+	rows, err := cache.db.Query(cache.db.Rebind(`
+		SELECT id FROM nodes
+		WHERE last_contact_success >= ? AND last_contact_success >= last_contact_failure`),
```

Comment:
Dont we need also the disqualified flag here `disqualified IS NULL`?

Reply:
Actually with the comment below, this would need to be restructured to include a flag if reliable or not.

Reply:
Yeah, for some reason I was thinking we disabled that check... but I think we just reset the disqualification statuses.

Reply:
seems like this also might have fixed the intermittent failures